src/CommentForm.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react'
+import { postComment } from './utils/api'
+const hardcodedUsername = 'cooljmessy'
+
+function CommentForm({ article_id, setComments, setCommentSubmitted, setCommentButtonClick }) {
+  const [username, setUsername] = useState(hardcodedUsername)
+  const [body, setBody] = useState('')
+  const [error, setError] = useState(null)
+
```

Comment:
As the comment updating is reliant upon an api call, we should have a loading pattern here